Projects/TEAM-3/Mykhailenko/QA2022_04_320/ex320.Tests/FigureTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace ex320.Tests
+{
+    [TestFixture]
+    public class FigureTests
+    {
+        /*
+        Directly methods of class Figure, I can only create objects of classes of heirs.
```

Comment:
It is Ok. It doesn't matter how you are calling the method. The main goal is to test the algorithms in this method.